arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -509,13 +558,12 @@
           "maxLength": 24
         },
         "type": {
-          "description": "Specifies the type of the resource.",

```

Comment:
Returned the property. http://editor.swagger.io/#/ reports that the extra reference is ignored.